lib/layouts/MenuBar.js

```diff
@@ -35,9 +35,9 @@ const Search = (props) => (
   />
 );
 
-const SubTitle = ({ iconProps, children }) => (
+const SubTitle = ({ iconProps, children, ...otherProps }) => (
   <h6
-    data-cy="menubar-subtitle-heading"
+    data-cy={otherProps["data-cy"] || "menubar-subtitle-heading"}
```

Comment:
@datt _a Same `otherProps["data-cy"]` is provided for Menubar, subTitle and Item. How to differentiate b/w them.